repo/meta/schema/v3-resource-schema.json

```diff
@@ -141,15 +154,18 @@
                 }
               },
               "additionalProperties": false,
-              "type": "object"
+              "type": "object",
+              "description": "Schema for MacOS (darwin) CLI."
             }
           },
           "additionalProperties": false,
-          "type": "object"
+          "type": "object",
+          "description": "Enumerates all binaries available in this package."
         }
       },
       "additionalProperties": false,
-      "type": "object"
+      "type": "object",
+      "description": "Object describing all externally hosted assets this package will need."
```

Comment:
Suggest: `Object that describes all of the required externally hosted assets for this package.`